Model/Navision/Order.php

```diff
@@ -347,14 +356,21 @@ protected function addItem(\Magento\Sales\Api\Data\OrderItemInterface $item, &$r
             )
             || $item->getProductType() == ProductType::TYPE_VIRTUAL
         ) {
+            $websiteId = $this->storeManager->getStore($item->getStoreId())->getWebsiteId();
             $child = $root->addChild('order_item');
 
             $child->mag_item_id = $item->getSku();
             $child->name = $item->getName();
             $child->quantity = $item->getQtyOrdered();
-            $child->unit_price = ($item->getParentItem() && ($item->getParentItem()->getProductType() != \Magento\Catalog\Model\Product\Type::TYPE_BUNDLE))
-                ? $item->getParentItem()->getBasePrice()
-                : $item->getBasePrice();
+            if ($item->getParentItem() && ($item->getParentItem()->getProductType() != ProductType::TYPE_BUNDLE)) {
```

Comment:
Send base/not base amounts not only for `unit_price` but also for `line_discount_amount`